### ISSUE TITLE
Use in Google Colab

### ISSUE THREAD

**Issue by fedhits** [grande-dev/Augmented-Neural-Lyapunov-Control#6]
Thank you for providing a valuable code.

While trying to use the repo in Google Colab, I get this error:

  File "/usr/local/lib/python3.8/site-packages/dreal/__init__.py", line 6, in <module>
    from dreal._dreal_py import *
ImportError: /usr/local/lib/python3.8/site-packages/dreal/../../../libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /opt/libibex/2.7.4/lib/libibex.so)

The error message indicates that the version of GLIBCXX (GNU C++ Library) required by the libibex.so library is not found in the Colab environment. How could this be solved?

Thank you very much again

**Comment by grande-dev** [comment]
@fedhits thank you for your interest in our software tool.

Unfortunately, I cannot recall encountering this error before as we never attempted to run the code on Google Colab. As a first test, I would suggest ruling out potential incompatibilities between the imported version of dReal and Python3.8. 
The two installation files provide packages and versions compatible with Python3.7 ([ANLC_v2/requirements.txt](https://github.com/grande-dev/Augmented-Neural-Lyapunov-Control/blob/master/ANLC_v2/requirements.txt)) and with Python3.9 ([ANLC_v2/requirements_v39.txt](https://github.com/grande-dev/Augmented-Neural-Lyapunov-Control/blob/master/ANLC_v2/requirements_v39.txt)). 

Have you already tried to run the code with either one of these two Python releases? If you have, did you experience any similar error that you can report?

**Comment by fedhits** [comment]
Yes, I have tried them both, but unfortunately the error persists

**Comment by grande-dev** [comment]
Thank you for ruling out this possibility.
We did not initially intend to run the tool on Google Colab, but I will have a look into this option in the following days and let you know if the attempt is successful.
